packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -89,6 +89,38 @@ export class MyAlgoWalletSession {
 		}
 	}
 
+	/**
+	 * @async
+	 * @description Sign a teal program (https://algorand.github.io/js-algorand-sdk/modules.html#tealSign)
+	 * @param logic Teal program
+	 * @param address Signer Address
+	 * @returns Returns signed teal
+	 * for more info: https://connect.myalgo.com/docs/interactive-examples/TealSign
+	 */
+	async signLogicSigUsingTeal(logic: string | Uint8Array, address: string): Promise<Uint8Array> {
```

Comment:
should we just name it signLogicSIgnature?

Reply:
okay will update